geps/gep-1713/index.md

```diff
@@ -57,11 +60,38 @@ once the API is gratuated to stable it will be renamed to `ListenerSet`.
 ```go
 type GatewaySpec struct {
 	...
+	// AllowedListeners defines which ListenerSets can be attached to this Gateway.
+	// While this feature is experimental, the default value is to allow no ListenerSets.
+	//
 	AllowedListeners *AllowedListeners `json:"allowedListeners"`
+}
+
+type GatewayStatus struct {
 	...
+	// AttachedListeners represents the total number of ListenerSets that have been
+	// successfully attached to this Gateway.
+	//
+	// Successful attachment of a ListenerSets to a Gateway is based solely on the
+	// combination of the AllowedListeners field on the Gateway
+	// and the ListenerSets's ParentRefs field. A ListenerSets is successfully attached to
+	// a Gateway when it is selected by the ListenerSets's AllowedListeners field
+	// AND the ListenerSets has a valid ParentRef selecting the Gateway
+	// resource as a parent resource. ListenerSets status does not impact
```

Comment:
Uh, where did this come from? This is the complete opposite of how AttachedRoutes works, and basically defeats the point of having this number at all.

As described, this is basically "count of Listeners/ListenerSets that refer to this Gateway" rather than "count of ListenerSets that have successfully attached to the Gateway". The point here is to track the Listeners that have actually attached and are relevant, I thought.